tests/utils/tools.php

```diff
@@ -88,6 +88,15 @@ function get_module_info($row)
     return $matches[1];
 }
 
+function create_test_manager(string $uri = null, array $options = [], array $driverOptions = [])
+{
+    if (getenv('API_VERSION') && ! isset($driverOptions['serverApi'])) {
+        $driverOptions['serverApi'] = new \MongoDB\Driver\ServerApi(getenv('API_VERSION'));
+    }
+
+    return new MongoDB\Driver\Manager($uri ?? URI, $options, $driverOptions);
```

Comment:
Top of the file has `use MongoDB\Driver\Manager;` so this can just be Manager.

Reply:
You are correct...the joy of not having PHP code assist in my IDE.